Formula/drush-launcher.rb

```diff
@@ -0,0 +1,15 @@
+class DrushLauncher < Formula
+  desc "Small wrapper around Drush for your global $PATH"
+  homepage "https://github.com/drush-ops/drush-launcher"
+  url "https://github.com/drush-ops/drush-launcher/releases/download/0.9.1/drush.phar"
+  sha256 "52b5e22031789e087e4f79ac49c74d94c0379ad6859f3ec14671ce1647a3b7a6"
+  license "GPL-2.0-or-later"
+
+  def install
+    bin.install "drush.phar" => "drush"
+  end
+
+  test do
+    system "#{bin}/drush", "--drush-launcher-version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
That is not so simple in this case. Because drush-launcher doesn't actually have any features of its own, but passes on all commands to a Drupal installation with drush.

Reply:
Can we attempt to access a non-existent Drupal installation and check we get the expected error message and exit code?